### ISSUE TITLE
Docs show incorrect flag names

### ISSUE THREAD

**Issue by qrilka** [commercialhaskell/stack#2128]
I see
```
--enable-executable-profiling --enable-library-profiling
```
at http://docs.haskellstack.org/en/stable/GUIDE/#debugging and stack (I have version 1.0.2) says:
```
Invalid option `--enable-executable-profiling'
```
Removing "enable" makes it work but according to #2074 even `--profile` should be enough

**Comment by mgsloan** [comment]
Good point!  Documentation improvement PRs appreciated.

**Comment by mgsloan** [comment]
Fixed by @sjakobi in #2398
